test/new-e2e/scenarios/system-probe/main.go

```diff
@@ -3,6 +3,9 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
+//go:build !windows
+// +build !windows
```

Comment:
FYI you don't need the old format anymore

Reply:
Which one is old?